docs/design/autoware-interfaces/ad-api/features/vehicle-status.md

```diff
@@ -0,0 +1,28 @@
+# Vehicle status
+
+## Related API
+
+- {{ link_ad_api('/api/vehicle/kinematics') }}
+- {{ link_ad_api('/api/vehicle/status') }}
+- {{ link_ad_api('/api/vehicle/dimensions') }}
+- {{ link_ad_api('/api/vehicle/doors/layout') }}
+- {{ link_ad_api('/api/vehicle/doors/status') }}
+
+## Kinematics
+
+This is an estimate of the vehicle kinematics. The vehicle position is necessary for applications to schedule dispatches.
+Also, using velocity and acceleration, applications can find vehicles that need operator assistance, such as stuck or brake suddenly.
+
+## Status
+
+This is the status provided by the vehicle. The indicators and steering are mainly used for visualization and remote control.
+The remaining energy can be also used for vehicle scheduling.
+
+## Doors
+
+This feature is available if the vehicle provides a software interface for the doors.
+It can be used to create user interfaces for passengers or to control sequences at bus stops.
+
+## Dimensions
```

Comment:
Were there any discussion somewhere if we want to have this information shared via topic or service? 

These are static information, and I was wondering if we want to distribute this as AD API. For example, we are not sharing map through topic in AD API. even though it is probably used by visualization tools. (obviously data size is very different though)

Reply:
> Were there any discussion somewhere if we want to have this information shared via topic or service?

@mitsudome-r I heard it's used for visualization in vehicle operation tools. Wheelbase is mentioned in https://github.com/autowarefoundation/autoware/issues/3226. After looking at the details, it is used to visualize the vehicle head at stop position.

> These are static information, and I was wondering if we want to distribute this as AD API. For example, we are not sharing map through topic in AD API. even though it is probably used by visualization tools. (obviously data size is very different though)

The visualization issue https://github.com/autowarefoundation/autoware/issues/3225 also mentions map and I think it should be added as AD API as well. Transient local topics and service are one option for providing static information. If other methods like parameter and setting file are used, it should be defined as AD API. Considering applications running on different ECU, the configuration files are cumbersome and do not need to be parameters as they are not updated. Therefore, I think it's easier to follow the methods used by other API.

Reply:
@mitsudome-r Would it be better to drop the dimensions from this PR for discussion about static information?

Reply:
If it makes easier for you. I don't mind splitting the PR.